### ISSUE TITLE
Failed to build ldms-plugins-llnl with LDMS on alternative path

### ISSUE THREAD

**Issue by weijianwen** [LLNL/ldms-plugins-llnl#7]
Hi,

I followed instructions on #3 but failed to build ldms-plugins-llnl with LDMS installed on an alternative path, say /opt/ovis. Steps to reproduce this issue:

```
./configure --prefix=/opt/ovis --with-libldms-prefix=/opt/ovis
...
checking for libldms... no
configure: error: libldms of ldms/ldms.h not found
```

But LDMS has already installed on /opt/ovis.

```
[root@cas005 ldms-plugins-llnl]# ll /opt/ovis/include/ldms/ldms.h
-rw-r--r-- 1 root root 65727 Dec  1 22:59 /opt/ovis/include/ldms/ldms.h
```

I appreciate if any suggestion is given.

Thank you!

**Comment by morrone** [comment]
Try the latest release, 1.11. I believe that I fixed the --with-libldms-prefix option.

**Comment by weijianwen** [comment]
Thank you @morrone . This issue has been resolved on 1.11.
